Code/src/main/java/nl/utwente/group10/ui/components/OutputBlock.java

```diff
@@ -0,0 +1,67 @@
+package nl.utwente.group10.ui.components;
+
+import javafx.fxml.FXMLLoader;
+import javafx.scene.control.Label;
+import nl.utwente.ewi.caes.tactilefx.fxml.TactileBuilderFactory;
+
+import java.io.IOException;
+
+/**
+ * OutputBlock is an extension of Block that only provides a display of the input
+ * it receives through it's inputAnchor.
+ * The input will be rendered visually on the Block.
+ * OutputBlock can be empty and contain no value at all, the value can be altered at any time
+ * by providing a different input source using a Connection.
+ */
+public class OutputBlock extends Block {
+	
+	/** The input this Block is receiving.**/
+	private String input;
```

Comment:
This seems to be a push-based model: you apparently expect functions to *push* their own results into all connected output blocks. This is reasonable, but it might be easier to have a pull-based model, where blocks like these *pull* their values from their 'parents' when they need them. This makes invalidation slightly more straightforward, and means that FunctionBlock does not need to know about OutputBlock.